app/assets/stylesheets/_aleph.scss

```diff
@@ -11,3 +11,56 @@
   margin-right: .1rem;
  color: $error;
 }
+
+.discovery-full-record-availability-info {
+
+  .list-local-locations {
+    list-style-type: none;
+    margin-top: 1rem;
+    padding-left: 0;
+    font-size: 1.4rem;
+  }
+
+  .result-local-location {
+    margin-bottom: 0;
+    border-top: 1px solid $gray-l2;
+    padding: .5rem 1rem;
+  }
+
+  .location-wrap, 
+  .availability-wrap {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: .5rem;
+  }
+  
+  .location {
+
```

Comment:
There's a few empty classes in this file. Are they for good reasons?

Reply:
yes, laziness.